Makefile

```diff
@@ -0,0 +1,11 @@
+
+default: build
```

Comment:
So a `.PHONY` target should be added, to express the fact that Make should not have to search for the files `default`, `build`, `dependencies` or `image`.

The target can be put wherever you want it and shall look like this:

```
.PHONY: default build dependencies image
```

Ref. https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html